pkgs/applications/video/screenkey/default.nix

```diff
@@ -1,50 +1,60 @@
 { lib
 , substituteAll
-, buildPythonApplication
-, fetchFromGitHub
-, distutils_extra
-, setuptools-git
+, fetchFromGitLab
+# native
 , intltool
-, pygtk
-, libX11
-, libXtst
 , wrapGAppsHook
-, gnome3
+, file
+# not native
+, xorg
+, gobject-introspection
+, gtk3
+, python3
 }:
-buildPythonApplication rec {
+
+python3.pkgs.buildPythonApplication rec {
   pname = "screenkey";
-  version = "0.9";
+  version = "1.2";
 
-  src = fetchFromGitHub {
-    owner = "wavexx";
+  src = fetchFromGitLab {
+    owner = "screenkey";
     repo = "screenkey";
-    rev = "screenkey-${version}";
-    sha256 = "14g7fiv9n7m03djwz1pp5034pffi87ssvss9bc1q8vq0ksn23vrw";
+    rev = "v${version}";
+    sha256 = "1x13n57iy2pg3h3r994q3g5nbmh2gwk3qidmmcv0g7qa89n2gwbj";
   };
 
-  patches = [
-    (substituteAll {
-      src = ./paths.patch;
-      inherit libX11 libXtst;
-    })
-  ];
-
   nativeBuildInputs = [
-    distutils_extra
-    setuptools-git
+    # Shouldn't be needed once https://gitlab.com/screenkey/screenkey/-/issues/122 is fixed.
     intltool
-
+    file
```

Comment:
What is `file` used for?

Reply:
It would be nice to find out what uses the file so we can document it in the comment.

Reply:
I had a slight feeling that it's related to `distutils_extra`, and I searched via the web interface at https://bazaar.launchpad.net/~registry/python-distutils-extra/debian/files but I couldn't find any evidence of this to be the cause :/. Perhaps we can hope that my comment mentioning https://gitlab.com/screenkey/screenkey/-/issues/122 is good enough?

Reply:
OK I added a comment stating that we don't know why file is needed, but without it there are "command not found" errors.